app/src/components/HomePage/HomePage.tsx

```diff
@@ -158,60 +170,94 @@ export default function HomePage({
+            className="h-full pt-[48px] pb-[100px]"
             bg="background.backgroundLight"
             px={8}
           >
             <Box className="mx-auto max-w-full w-[1090px] css-0">
-              <Text
-                color="content.primary"
-                fontWeight="bold"
-                lineHeight="24px"
-                fontSize="headline.sm"
-                fontFamily="heading"
-                fontStyle="normal"
-              ></Text>
+              {/* Years section */}
               {!isPublic ? (
-                <Tabs align="start" variant="line" isLazy>
-                  <TabList>
-                    {[
-                      t("tab-emission-inventory-calculation-title"),
-                      t("tab-emission-inventory-results-title"),
-                    ]?.map((tab, index) => (
-                      <Tab key={index}>
-                        <Text
-                          fontFamily="heading"
-                          fontSize="title.md"
-                          fontWeight="medium"
-                        >
-                          {t(tab)}
-                        </Text>
-                      </Tab>
-                    ))}
-                  </TabList>
-                  <TabPanels>
-                    <TabPanel>
-                      <InventoryCalculationTab
-                        lng={lng}
-                        inventory={inventory}
-                        inventoryProgress={inventoryProgress}
-                        isUserInfoLoading={isUserInfoLoading}
-                        isInventoryProgressLoading={isInventoryProgressLoading}
-                      />
-                    </TabPanel>
-                    <TabPanel>
-                      <InventoryReportTab
-                        isPublic={isPublic}
-                        lng={lng}
-                        population={population}
-                        inventory={inventory}
-                        inventoryProgress={inventoryProgress}
-                        isUserInfoLoading={isUserInfoLoading}
-                        isInventoryProgressLoading={isInventoryProgressLoading}
-                      />
-                    </TabPanel>
-                  </TabPanels>
-                </Tabs>
+                <>
+                  <Box className="w-full mb-6 flex items-center justify-between">
+                    <Text
+                      color="content.primary"
+                      fontWeight="bold"
+                      lineHeight="24px"
+                      fontSize="headline.sm"
+                      fontFamily="heading"
+                      fontStyle="normal"
+                    >
+                      {t("inventory-year")}
+                    </Text>
+                    <Button
+                      data-testid="add-new-inventory-button"
+                      title={t("add-new-inventory")}
+                      leftIcon={<AddIcon h="16px" w="16px" />}
+                      h="48px"
```

Comment:
Do we have to hard code the height or could we just add a certain padding and then make the size depend on the icon and text?